Formula/gcc.rb

```diff
@@ -210,7 +208,7 @@ def post_install
       #   * `-L#{HOMEBREW_PREFIX}/lib` instructs gcc to find the rest
       #     brew libraries.
       # Note: *link will silently add #{libdir} first to the RPATH
-      libdir = HOMEBREW_PREFIX/"lib/gcc/#{version_suffix}"
+      libdir = HOMEBREW_PREFIX/"lib/gcc/current"
```

Comment:
@iMichka @danielnachun I don't know what this does, so would be good to check that this makes sense.

Reply:
We can clean this up in a second step as this is in the postinstall block.

`current` looks ok to me given the other changes to gcc made here.
There are some explanations above: the spec file just makes sure the right `-L` flags are passed when using brewed gcc (without shims, so when you want to compile your own stuff locally). With this our users don't need to set the `-L` flags manually for the internal gcc libraries.

As to why not using `opt_lib`: why not, might be worth a try.